web/backend/readme.md

```diff
@@ -4,21 +4,9 @@ Once the project cloned type `npm install` to install the packages.
 
 # Config
 
-The project contains one file that is not in git (because it is in the .gitignore).
-This file is called `config.json` and is located at the root of the express project.
-Please use the `config.json.template` to start with.
-
-This files contains all the secrets and also the running information. It should be formatted this way :
-
-```json
-{
-  "FRONT_END_URL" : "<url of the current site, this is used for the tequila callback>",
-  "DELA_NODE_URL" : "<url of the dela node>",
-  "SESSION_SECRET" : "<session secret>",
-  "PUBLIC_KEY" : "<public key>",
-  "PRIVATE_KEY" : "<private key>"
```

Comment:
is this documentation about the different fields found in `config.env.template` ?

Reply:
Documentation added.